app/actions.py

```diff
@@ -487,7 +487,9 @@ def cursorSelectPageDown(self):
     Returns:
       None.
     """
-    if self.selectionMode == app.selectable.kSelectionNone:
+    # Selection mode must be kSelectionCharacter
+    # or else weird behaviors will occur.
+    if self.selectionMode != app.selectable.kSelectionCharacter:
       self.doSelectionMode(app.selectable.kSelectionCharacter)
```

Comment:
I think the comment is more a note to me in this review, so let's remove it from the code/PR.
Having 
if foo != X:
  foo = X
is the same as writing
foo = X
Please go ahead and remove the 'if' line.

Reply:
I was thinking that by having this check, it would save some time since calling doSelectionMode seems very expensive so the if statement was more of a performance decision. 

I do think, however, that I should change the call to self.selectionCharacter(). 

Do you still want me to delete the if statement?